hathor/transaction/storage/remote_storage.py

```diff
@@ -8,6 +8,8 @@
 from twisted.logger import Logger
 
 from hathor import protos
+print('#### ')
+print(protos)
```

Comment:
I think these are leftovers from testing.

Reply:
Yeah, this was remove in the last commit, thanks!